source/features/clean-repo-filelist-actions.tsx

```diff
@@ -34,19 +34,20 @@ function cleanFilelistActions(searchButton: Element): void {
 		addTooltipToSummary(addFileDropdown, 'Add file');
 	}
 
-	const codeDropdownButton = select('get-repo summary');
-	if (codeDropdownButton) { // This dropdown doesn't appear on `isSingleFile`
-		addTooltipToSummary(codeDropdownButton, 'Clone, open or download');
-
-		// Users with Codespaces enabled already have an icon in the button https://github.com/refined-github/refined-github/pull/5074#issuecomment-983251719
-		const codeIcon = select('.octicon-code', codeDropdownButton);
-		if (codeIcon) {
-			removeTextNodeContaining(codeIcon.nextSibling!, 'Code');
-		} else {
-			removeTextNodeContaining(codeDropdownButton.firstChild!, 'Code');
-			codeDropdownButton.prepend(<CodeIcon/>);
-		}
+	if (!pageDetect.isRepoRoot()) {
+		return;
 	}
+
+	const codeDropdownButton = select('get-repo summary')!;
+	addTooltipToSummary(codeDropdownButton, 'Clone, open or download');
+
+	const label = select('.Button-label', codeDropdownButton)!;
+	if (!select.exists('.octicon-code', codeDropdownButton)) {
+		// The icon is missing for users without Codespaces https://github.com/refined-github/refined-github/pull/5074#issuecomment-983251719
```

Comment:
I don't know if this is still applicable since Codespaces are generally available now. I only tested this code in the logged out view (which is actually unsupported by isRepoRoot())